ssn/ssn_separated/ssn.ttl

```diff
@@ -98,7 +97,7 @@ rdfs:seeAlso rdf:type owl:AnnotationProperty .
 ###  http://www.w3.org/ns/ssn/deployedOnPlatform
 ssn:deployedOnPlatform rdf:type owl:ObjectProperty ;
                        owl:inverseOf ssn:inDeployment ;
-                       rdfs:comment "Relation between a deployment and the platform on which the system is deployed." ;
+                       rdfs:comment "Relation between a Deployment and the Platform on which the system is deployed." ;
                        rdfs:isDefinedBy "http://www.w3.org/ns/ssn" ;
                        rdfs:label "deployed on platform" ;
                        rdfs:seeAlso "http://www.w3.org/2005/Incubator/ssn/wiki/SSN_Deploy#Deployment" .
```

Comment:
Yes, I kept only the ones were I thought we haven't discussed the concepts yet. But, we can delete them all, agree.